src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -3,7 +3,7 @@
 
 use std::io::{self, stdin};
 use std::result;
-use std::sync::{Arc, Mutex};
+use std::sync::{Arc, Barrier, Mutex};
```

Comment:
It is not really straightforward to reproduce the problem. It might actually have an impact when booting a Linux kernel. I'll have to investigate this more.